web/README.md

```diff
@@ -0,0 +1,418 @@
+# HarbourBridge: Web APIs (Experimental)
+
+HarbourBridge is a stand-alone open source tool for Cloud Spanner evaluation,
+using data from an existing PostgreSQL or MySQL database. This README provides
+details of the tool's Web APIs capabilities, Web APIs allows user to visualize the conversion
+using UI and interact with the tool to customize some of the conversion. For general HarbourBridge information
+see this [README](https://github.com/cloudspannerecosystem/harbourbridge#harbourbridge-turnkey-spanner-evaluation). Note that this feature is experimental, and should not be used for production database.
+
+### Starting web server for HarbourBridge
+
+The following example assume `harbourbridge` has been added to your PATH
+environment variable.
+
+HarbourBridge's Web API feature can be used with all the driver modes available, using mysql or postgres dump or direct connection.
+
+To start HarbourBridge web server, run:
+
+```sh
+harbourbridge --web
+```
+
+The tool will be available on port 8080
+
+You can go to `<base-url>/` to visit the UI of the tool.
+
+    eg: localhost:8080/
+
+For more details on how to use the UI, you can visit: `<base-url>/userManual.html`.
+
+    eg: localhost:8080/userManual.html
+
+<ins>**Note:**</ins>
+
+Data migration is not supported when using `pg_dump` or `mysqldump` driver options when the converted schema contains Interleaved tables. It is recommended to use `postgres` or `mysql` driver for data migration with schema having Interleaved tables.
+
+## APIs
+
+These are the REST APIs and their details:
+
+### Connect
+
+`/connect` is a POST API which is used in case of direct connection. It takes driver config in request body, following is the json structure for the same.
+
+#### Method
+
+`POST`
+
+#### Request body
+
+Connection information along with driver name.
+
+Example
+
+```json
+{
+  "Driver": "postgres",
+  "Host": "localhost",
+  "Port": "5432",
+  "Database": "dbname",
+  "User": "user",
+  "Password": "password"
+}
+```
+
+#### Response body
+
+No response body is returned.
+
+### Convert
+
+(1) `/convert/infoschema` is a GET API followed by `/connect` API to convert using infoschema mode. It returns the schema conversion state in json format.
```

Comment:
This seems to indicate that `/convert/infoschema` followed by `/connect`?
Isn't that the other way around: you do connect first and then convert?

Reply:
Yes, first we make database connection using /connect, then we just call GET API /convert/infoschema to process conversion.